ui/proxy/js/components/IncidentProgressionPanel.react.js

```diff
@@ -74,19 +74,21 @@ var IncidentProgressionPanel = React.createClass({
                                                     .range([d3.hsl(270, .75, .35), d3.hsl(70, 1.5, .8)])
                                                     .interpolate(d3Interpolate.interpolateCubehelix);
         }
-
-        return;
     },
     draw: function () {
-        var requestsRoot, referedRoot, leaves, node, canvasWidth, canvasHeight, width, height;
+        var requestRoot, referedRoot, node, canvasWidth, width, height, nodes, links;
 
-        // Build Requests root node
-        requestsRoot = d3.hierarchy({
-            id: this.state.root.id,
-            name: '',
-            type: this.state.root.type,
-            children: this.state.root.requests
-        });
+        /*leaves = Math.max(requestsRoot.leaves().length, referedRoot.leaves().length);
```

Comment:
Is there a particular reason to keep this code commented? 
If so, there is one /* starting and then in between a comment with // and finally a closing */. I would say treat everything as a big comment and remove the //. 

If it's not required I'd say remove the entire comment.